functional-tests/fn_bridge_deposit_reclaim_drt_seen.py

```diff
@@ -0,0 +1,230 @@
+                # This is 10 minutes.
+                # NOTE: Alternatively, we could take one bridge operator down
+                #       to prevent the DRT from being processed.
+                message_interval=int(10 * 60 * 1_000),
+            )
+        )
+        self.logger = get_logger("BridgeDepositReclaimDrtSeenTest")
+
+    def main(self, ctx: flexitest.RunContext):
+        btc = ctx.get_service("bitcoin")
+        seq = ctx.get_service("sequencer")
+        reth = ctx.get_service("reth")
+
+        seqrpc = seq.create_rpc()
+        btcrpc: BitcoindClient = btc.create_rpc()
+        rethrpc = reth.create_rpc()
+
+        btc_url = btcrpc.base_url
+        btc_user = btc.props["rpc_user"]
+        btc_password = btc.props["rpc_password"]
+
+        self.logger.debug(f"BTC URL: {btc_url}")
+        self.logger.debug(f"BTC user: {btc_user}")
+        self.logger.debug(f"BTC password: {btc_password}")
+
+        bridge_pk = get_bridge_pubkey(seqrpc)
+        self.logger.debug(f"Bridge pubkey: {bridge_pk}")
+        el_address = "deadf001900dca3ebeefdeadf001900dca3ebeef"
+        self.logger.debug(f"EL address: {el_address}")
+        addr = get_address(0)
+        self.logger.debug(f"Address: {addr}")
+        refund_addr = get_address(1)
+        self.logger.debug(f"Refund address: {refund_addr}")
+        recovery_addr = get_recovery_address(0, bridge_pk)
+        self.logger.debug(f"Recovery Address: {recovery_addr}")
+
+        seq_addr = seq.get_prop("address")
+        self.logger.debug(f"Sequencer Address: {seq_addr}")
+
+        # First let's see if the EVM side has no funds
+        # Make sure that the el_address has zero balance
+        balance = int(rethrpc.eth_getBalance(f"0x{el_address}"), 16)
+        assert balance == 0, "EVM balance is not zero"
+
+        # Also make sure that the all addresses have zero balance
+        btc_addr_balance = get_balance(
+            addr,
+            btc_url,
+            btc_user,
+            btc_password,
+        )
+        self.logger.debug(f"Address BTC balance (before DRT): {btc_addr_balance}")
+        assert btc_addr_balance == 0, "Address BTC balance is not zero"
+        btc_refund_balance = get_balance(
+            refund_addr,
+            btc_url,
+            btc_user,
+            btc_password,
+        )
+        self.logger.debug(f"Refund BTC balance (before DRT): {btc_refund_balance}")
+        assert btc_refund_balance == 0, "Refund BTC balance is not zero"
+        btc_recovery_balance = get_balance_recovery(
+            recovery_addr,
+            bridge_pk,
+            btc_url,
+            btc_user,
+            btc_password,
+        )
+        self.logger.debug(f"DRT BTC balance (before DRT): {btc_recovery_balance}")
+        assert btc_recovery_balance == 0, "Recovery BTC balance is not zero"
+
+        # Generate Plenty of BTC to address for the DRT
+        self.logger.debug(f"Generating 102 blocks to address: {addr}")
```

Comment:
feel free to use the "funded addresses" functionality, if it can help here. #467

Reply:
Done in 421a23dca071bcb1e198b18fb3ec6adbab752a7f